platform/android/app/src/main/kotlin/coop/polypoly/polypod/homescreen/HomeScreenFragment.kt

```diff
@@ -169,13 +175,14 @@ fun Screen(screen: Screen) {
         ),
         modifier = Modifier
             .width(screen.layout.width)
+            .background(Color(0xFFEDF2F7))
```

Comment:
nit: the background-color could be part of `screen.layout` at a future iteration maybe? 🤔

Reply:
Even in this iteration, literals are not a good thing; a named constant might be a better alternative.